docs/advanced/patterns_by_usecase.md

```diff
@@ -632,39 +632,41 @@ The above example does not work as we are not checking the value of `event.value
   <summary>
   Discriminated Unions in TypeScript can also work with hook dependencies in React. The type matched is automatically updated when the corresponding union member based on which a hook depends, changes. Expand more to see an example usecase.
   </summary>
- 
```

Comment:
this stray whitespace causes the bug in mdx to not regocnize ````tsx` as markdown syntax

Reply:
so it's the known mdx issue not a docusaurus bug right?

Reply:
yes, sorry i thought it was an issue with docusaurus/mdx-loader